indexer/services/comlink/src/types.ts

```diff
@@ -666,3 +666,32 @@ export interface VaultPosition {
 export interface MegavaultPositionResponse {
   positions: VaultPosition[];
 }
+
+/* ------- Affiliates Types ------- */
+export interface AffiliateReferralCodeRequest{
+  address: string
+}
+
+export interface AffiliateReferralCodeResponse {
+  referralCode: string | null,
+}
+
+export interface AffiliateAddressResponse {
```

Comment:
Need to add corresponding Request type for this

Reply:
Will add in upcoming stubs